Tools/Dev/Postgres/docker-compose.yml

```diff
@@ -10,4 +10,7 @@ services:
     restart: unless-stopped
     volumes:
       - ./backups:/backups
-      - ./pgdata:/var/lib/postgresql/data
```

Comment:
Why did this get changed?  If you want the external folder to be mapped into data then we can make that change but the internal path should remain at /var/lib/postgresql otherwise we won't be externalizing the data.

Reply:
Without this change postgress wont start due to permission issues.